agent/consul/state/acl.go

```diff
@@ -884,18 +884,18 @@ func aclPolicySetTxn(tx WriteTxn, idx uint64, policy *structs.ACLPolicy) error {
 	}
 
 	if existing != nil {
-		if policy.ID == structs.ACLPolicyGlobalManagementID {
+		if builtinPolicy, ok := structs.ACLBuiltinPolicies[policy.ID]; ok {
 			// Only the name and description are modifiable
 			// Here we specifically check that the rules on the global management policy
```

Comment:
```suggestion
			// Here we specifically check that the rules on the builtin policy
```
I think just a comment to update